SPECS/in-band-manageability/in-band-manageability.spec

```diff
@@ -0,0 +1,303 @@
+# Macros needed by SELinux
+%global selinuxtype targeted
+%global debug_package %{nil}
+
+Summary:        An agent to manage systems via in-band connection
+Name:           in-band-manageability
+Version:        1.0.0
+Release:        1%{?dist}
+Distribution:   Edge Microvisor Toolkit
+Vendor:         Intel Corporation
+License:        Apache-2.0
+URL:            https://github.com/open-edge-platform/edge-node-agents
+Source0:        %{url}/archive/refs/tags/%{name}/v%{version}.tar.gz#/%{name}-%{version}.tar.gz
+Source1:        intel_manageability.conf
+Source2:        inbm-configuration-replace-FQDN.sh
+Source3:        inbm.te
+Source4:        inbm.fc
+BuildRequires:  golang <= 1.24.7
+BuildRequires:  systemd-rpm-macros
+BuildRequires:  selinux-policy-devel
+BuildRequires:  make
+BuildRequires:  git
```

Comment:
we need to obsolete inbm, since these provide same functions and this new rpm replace the old inbm.

Obsoletes:      inbm

Reply:
done added. Thanks :).